testnets/cosstestnet/chain.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "cosstestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "COSS Testnet",
+  "chain_id": "coss-testnet-1",
+  "bech32_prefix": "coss",
+  "daemon_name": "cossd",
+  "node_home": "$HOME/.cossd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ucgas",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ucgas"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/coss-inscription/coss",
+    "recommended_version": "0.1.0-testnet",
+    "compatible_versions": [
+      "0.1.0-testnet"
+    ],
+    "cosmos_sdk_version": "0.50.2",
+    "consensus": {
+      "type": "tendermint",
+      "version": "0.38.2"
+    },
+    "cosmwasm_enabled": false,
+    "genesis": {
+      "genesis_url": "https://github.com/coss-inscription/networks/blob/main/testnet/coss-testnet-1/genesis.json"
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "4bf62eb8e9d65bb86751529ca233df574891602f",
+        "address": "35.72.222.149:26656",
+        "provider": ""
```

Comment:
Can you please add provider?

Reply:
ok, i will.

Reply:
> Can you please add provider?

the provider has been added, please help review again and merge this pr. Thanks!